internal/rest/avi_obj_string_group.go

```diff
@@ -160,11 +173,28 @@ func (rest *RestOperations) AviStringGroupCacheAdd(rest_op *utils.RestOp, vsKey
 			Uuid: uuid}
 
 		// StringGroup should not have a checksum
-		stringgroup_cache_obj.CloudConfigCksum = lib.StringGroupChecksum(keyvalues, stringgroup_cache_obj.Description, nil, false)
+		stringgroup_cache_obj.CloudConfigCksum = lib.StringGroupChecksum(keyvalues, nil, nil, nil, false)
```

Comment:
Do we really want to set the third parameter nil here or setting it false also works ?

Reply:
All the params that are set as nil are either pointers of slice of pointers.